webapp/package.json

```diff
@@ -54,6 +54,7 @@
     "classnames": "2.5.1",
     "core-js": "3.36.0",
     "date-fns": "3.3.1",
+    "date-fns-tz": "3.1.3",
```

Comment:
The web app currently uses `moment-timezone` for this, but given Moment is legacy at this point, this seems like a better bet

Reply:
yeah, date fns tz works really well with date fns and the whole idea is to work with native Date objects, not a custom ones like Moment does.